### PR TITLE
Omnibar position: Onboarding messages

### PR DESCRIPTION
**Pull request by 0nko**
Task/Issue URL: https://app.asana.com/0/0/1208157864474375/f

### Steps to test this PR

- [ ] Clear the app storage and trigger onboarding
- [ ] Change the omnibar position to the bottom
- [ ] Go to “google.com"
- [ ] Notice that during onboarding steps, whenever there is the “hand” icon, it points to the bottom

### UI changes
| Before  | After |
| ------ | ----- |
image|image|

### CODE REVIEW

**Comment by 0nko** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4942?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4942** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4942?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4941** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4941?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4940** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4940?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4939** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4939?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4930** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4930?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4886** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4886?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4885** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4885?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @0nko and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: app/src/main/java/com/duckduckgo/app/cta/ui/Cta.kt [hunk 1]
```diff
@@ -214,8 +217,10 @@ sealed class OnboardingDaxDialogCta(
             val quantityString =
                 if (size == 0) {
                     context.resources.getQuantityString(R.plurals.onboardingTrackersBlockedZeroDialogDescription, trackersFiltered.size)
+                        .getStringWithCorrectFinger(settingsDataStore.omnibarPosition)
```
Comment (malmstein) [inline app/src/main/java/com/duckduckgo/app/cta/ui/Cta.kt:220]:
```suggestion
                        .getStringForOmnibarPosition(settingsDataStore.omnibarPosition)
```

File: app/src/main/java/com/duckduckgo/app/cta/ui/Cta.kt [hunk 2]
```diff
@@ -635,3 +642,10 @@ fun DaxCta.canSendShownPixel(): Boolean {
     val param = onboardingStore.onboardingDialogJourney?.split("-").orEmpty().toMutableList()
     return !(param.isNotEmpty() && param.any { it.split(":").firstOrNull().orEmpty() == ctaPixelParam })
 }
+
+fun String.getStringWithCorrectFinger(position: OmnibarPosition): String {
```
Comment (malmstein) [inline app/src/main/java/com/duckduckgo/app/cta/ui/Cta.kt:646]: this needs a test

**Review by malmstein — APPROVED** [review]
Works as expected, but please add a test for the new case.

File: app/src/main/java/com/duckduckgo/app/cta/ui/Cta.kt (hunk 2 above)
Comment (0nko) [reply app/src/main/java/com/duckduckgo/app/cta/ui/Cta.kt]: Added [here](https://github.com/duckduckgo/Android/pull/4942/commits/2dac278adec66ced784a241ca14f1318284fea39).

**Pull request merged**